Classes/Issues/Managing/IssueManagingExpansionModel.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  IssueManagingExpansionModel.swift
+//  Freetime
+//
+//  Created by Ryan Nystrom on 11/13/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+import IGListKit
+
+final class IssueManagingExpansionModel: ListDiffable {
+
+    let expanded: Bool
+
+    init(expanded: Bool) {
+        self.expanded = expanded
+    }
+
+    // MARK: ListDiffable
+
+    func diffIdentifier() -> NSObjectProtocol {
+        return "managing-model" as NSObjectProtocol
```

Comment:
Shouldn't this return a unique key?

Reply:
There's only one expansion button (the one that says "Manage")